ts/Extensions/Exporting/Exporting.ts

```diff
@@ -0,0 +1,1963 @@
+     *
+     * @function Highcharts#moveContainers
+     *
+     * @private
+     *
+     * @param {Highcharts.HTMLDOMElement} moveTo
+     *        Move target
+     * @return {void}
+     */
+    function moveContainers(this: Chart, moveTo: HTMLDOMElement): void {
+        const chart = this;
+        (
+            chart.fixedDiv ? // When scrollablePlotArea is active (#9533)
+                [chart.fixedDiv, chart.scrollingContainer as any] :
+                [chart.container]
+
+        ).forEach(function (div: HTMLDOMElement): void {
+            moveTo.appendChild(div);
+        });
+    }
+
+    /**
+     * Add update methods to handle chart.update and chart.exporting.update and
+     * chart.navigation.update. These must be added to the chart instance rather
+     * than the Chart prototype in order to use the chart instance inside the
+     * update function.
+     * @private
+     */
+    function onChartInit(
+        this: ChartComposition
+    ): void {
+        const chart = this,
+            /**
+             * @private
+             * @param {"exporting"|"navigation"} prop
+             *        Property name in option root
+             * @param {Highcharts.ExportingOptions|Highcharts.NavigationOptions} options
+             *        Options to update
+             * @param {boolean} [redraw=true]
+             *        Whether to redraw
+             * @return {void}
+             */
+            update = (
+                prop: ('exporting'|'navigation'),
+                options: (ExportingOptions|NavigationOptions),
+                redraw?: boolean
+            ): void => {
+                chart.isDirtyExporting = true;
+                merge(true, chart.options[prop], options);
+                if (pick(redraw, true)) {
+                    chart.redraw();
+                }
+            };
+
+        chart.exporting = {
+            update: function (
+                options: ExportingOptions,
+                redraw?: boolean
+            ): void {
+                update('exporting', options, redraw);
+            }
+        };
+
+        // Register update() method for navigation. Can not be set the same way
+        // as for exporting, because navigation options are shared with bindings
+        // which has separate update() logic.
+        chartNavigationMixin.addUpdate(
+            function (
+                options: NavigationOptions,
+                redraw?: boolean
+            ): void {
+                update('navigation', options, redraw);
+            },
+            chart
+        );
+    }
+
+    /**
+     * The post utility
+     *
+     * @private
+     * @function Highcharts.post
+     *
+     * @param {string} url
+     * Post URL
+     *
+     * @param {object} data
+     * Post data
+     *
+     * @param {Highcharts.Dictionary<string>} [formAttributes]
+     * Additional attributes for the post request
+     */
+    export function post(
+        this: Chart,
```

Comment:
I don't think the `post` function should be attached to the (already overcrowded) `Chart.prototype`. It is a generic utility function that doesn't have anything to do with a chart, which is also why it previously was attached to `Highcharts`.

Do we have an alternative solution to this? Putting it in Utilities would be the most logic solution, but then it would bloat the core.

Reply:
Would it be okay to move it to something similar to FormatUtilities, like HttpUtilities?

Reply:
Yes I suppose so. I guess the challenge is to make it work with those tests that override Highcharts.post. If we can rethink how we do those, the `post` function could also be local to the Exporting module.

Reply:
The idea of a post event on the chart came to our mind, before we decided with the solution in question.

Reply:
The features that are most related to `post` is the `ajax` and `getJSON` functions, both found in `ts/Extensions/Ajax.ts`. The cleanest solution, thematically, may be to add the `post` function there and rename the file to `HttpUtilities`.